enterprise/frontend/src/embedding-sdk/components/private/PublicComponentWrapper.tsx

```diff
@@ -0,0 +1,25 @@
+import { SdkError } from "embedding-sdk/components/private/SdkError";
+import { useSdkSelector } from "embedding-sdk/store";
+import { getLoginStatus } from "embedding-sdk/store/selectors";
+
+export const PublicComponentWrapper = ({
```

Comment:
I think the beauty of this is that we can set up some plugin system to allow users to change these states. In the future we should probably have some sort of SdkLoading component as well.

We'll most likely need to modify the LoadingAndErrorWrapper to use SdkError and (the future) SdkLoading components so they can be customizable as well